### PR TITLE
Fix sdist package

### PR DESCRIPTION
**Pull request by ulope**
`setuptools_scm` forcibly includes all files under version control into the sdist package, ignoring `MANIFEST.in` and `find_packages`.
This fixes this by replacing the `find_files` function with a dummy one (see `setup.py::EggInfo.__init__()`).

See pypa/setuptools_scm#190

Additionally this uncovered that we had two directories that were missing `__init__.py` files (thereby being turned into namespace packages). That lead to mypy and pylint not covering them. 
This is now also fixed here (including two bugs in migrations mypy found).

### CODE REVIEW

File: raiden/storage/migrations/v18_to_v19.py [hunk 1]
```diff
@@ -69,7 +69,7 @@ def _add_blockhash_to_state_changes(storage: SQLiteStorage, cache: BlockHashCach
             data = json.loads(state_change.data)
             assert "block_hash" not in data, "v18 state changes cant contain blockhash"
             record = BlockQueryAndUpdateRecord(
-                block_number=int(data["block_number"]),
+                block_number=BlockNumber(int(data["block_number"])),
```
Comment (rakanalh) [inline raiden/storage/migrations/v18_to_v19.py]: Here, it is sufficient to do `BlockNumber(data["BlockNumber"])`

Comment (rakanalh) [reply raiden/storage/migrations/v18_to_v19.py]: Below are other instances as well.

File: raiden/storage/migrations/v19_to_v20.py [hunk 2]
```diff
@@ -4,15 +4,22 @@
 from eth_utils import to_canonical_address
 from gevent.pool import Pool
 
+from raiden import raiden_service  # pylint: disable=unused-import
```
Comment (rakanalh) [inline raiden/storage/migrations/v19_to_v20.py]: can you please just import `RaidenService` class directly here?

Comment (rakanalh) [reply raiden/storage/migrations/v19_to_v20.py]: Also, this can be wrapped in `if TYPE_CHECKING, import RaidenService` since we're keeping the type "quoted" in the function declarations

File: raiden/storage/migrations/v18_to_v19.py (hunk 1 above)
Comment (palango) [reply raiden/storage/migrations/v18_to_v19.py]: Are you sure? It doesn't seem to convert it:

```
In [3]: BlockNumber('1')
Out[3]: '1'
```

Comment (ulope) [reply raiden/storage/migrations/v18_to_v19.py]: No it doesn't convert. The type aliases are an identity function at runtime. 
We talked about it in person. The `int()` needs to stay in.

File: raiden/storage/migrations/v21_to_v22.py [hunk 3]
```diff
@@ -236,16 +236,16 @@ def _add_canonical_identifier_to_statechanges(
             )
 
             if should_delete:
-                delete_state_changes.append(state_change_record.identifier)
+                delete_state_changes.append(state_change_record.state_change_identifier)
```
Comment (konradkonrad) [inline raiden/storage/migrations/v21_to_v22.py]: yeah, good find! I have to admit, I get confused by the naming in https://github.com/raiden-network/raiden/blob/master/raiden/storage/sqlite.py#L13-L27

imho, we should stick to `.identifier` for the unique id field and only use the qualifiers for the cross referenced ids (i.e. `EventRecord.state_change_identifier`)

File: raiden/storage/migrations/v19_to_v20.py [hunk 4]
```diff
@@ -175,10 +189,10 @@ def _add_onchain_locksroot_to_snapshot(
                 channel["our_state"]["onchain_locksroot"] = serialize_bytes(our_locksroot)
                 channel["partner_state"]["onchain_locksroot"] = serialize_bytes(partner_locksroot)
 
-    return json.dumps(snapshot, indent=4), snapshot_record.identifier
+    return json.dumps(snapshot, indent=4), snapshot_record.state_change_identifier
```
Comment (konradkonrad) [inline raiden/storage/migrations/v19_to_v20.py]: Not sure if this is what we want here: `snapshot_record.identifier` should/does exist and is the unique id for the affected snapshot. Why did you change it?

File: raiden/storage/migrations/v21_to_v22.py (hunk 3 above)
Comment (konradkonrad) [reply raiden/storage/migrations/v21_to_v22.py]: btw, this is not for this very PR, just a drive-by. I'll resolve and create an issue instead.

File: raiden/storage/migrations/v19_to_v20.py (hunk 4 above)
Comment (ulope) [reply raiden/storage/migrations/v19_to_v20.py]: Because `snapshot_record` is a `StateChangeRecord` (at least according to the function annotation) and therefore mypy (and PyCharm) complained.

File: raiden/storage/migrations/v21_to_v22.py (hunk 3 above)
Comment (ulope) [reply raiden/storage/migrations/v21_to_v22.py]: Yes, that sounds like a good idea.

File: raiden/storage/migrations/v19_to_v20.py (hunk 4 above)
Comment (konradkonrad) [reply raiden/storage/migrations/v19_to_v20.py]: yeah, the typing of the signature is incorrect then: https://github.com/raiden-network/raiden/pull/4074/files#diff-30896e13fdd21557ed5512f90093e729R196

Comment (ulope) [reply raiden/storage/migrations/v19_to_v20.py]: Ok makes sense. I changed it.
